appveyor.yml

```diff
@@ -1,26 +1,67 @@
 build: false
-shallow_clone: false
-platform: 'x86'
-clone_folder: C:\projects\better-reflection
+clone_depth: 50
+platform:
+  - x64
+environment:
+  matrix:
+    - dependencies: lowest
+      php_version: 7.1
+    - dependencies: highest
+      php_version: 7.1
 
+  project_directory: c:\projects\better-reflection
+  composer_directory: c:\tools\composer
+  composer_executable: c:\tools\composer\composer.phar
+  php_archive_directory: c:\tools\php-archive
+  php_directory: c:\tools\php
+matrix:
+  fast_finish: true
+clone_folder: c:\projects\better-reflection
+cache:
+  - c:\tools\composer
+  - c:\tools\php-archive
+  - '%LOCALAPPDATA%\Composer'
+init:
+  - ps: $Env:PATH = $Env:php_directory + ';' + $Env:composer_directory + ';' + $Env:PATH
+  - ps: $Env:ANSICON = '121x90 (121x90)'
```

Comment:
It looks better:
- [Without colors](https://ci.appveyor.com/project/Ocramius/betterreflection-4jx2w/build/1.0.18/job/80yeat88u2pm067q)
- [With colors](https://ci.appveyor.com/project/Ocramius/betterreflection-4jx2w/build/1.0.20/job/0n9a1n647fhbekik)